pom.client.xml

```diff
@@ -401,6 +401,10 @@
                 <title>Azure Storage - Blobs</title>
                 <packages>com.azure.storage.blob*</packages>
               </group>
+              <group>
+                <title>Azure Storage Blob - Batch</title>
+                <packages>com.azure.storage.blob.batch*</packages>
```

Comment:
Will this conflict with the regex above? It seems like blob* will match first before blob.batch*

Reply:
It might, mind writing me an issue to fix this after this PR goes in? It is blocking a few releases from going out.

Reply:
Hm, since this is the documentation that gets published with our release, it would be worthwhile to verify this works?